lib/config/options/index.ts

```diff
@@ -1499,6 +1499,38 @@ const options: RenovateOptions[] = [
     env: false,
     supportedPlatforms: ['github'],
   },
+  {
```

Comment:
I would like for us to implement #14427:
- New flag `experimental`
- Adds a Warning admonition to docs
- Notes that it can be changed or removed at any time (although we'll do our best not to)

Reply:
Is this a replacement for `unpulished`?